searx/settings.yml

```diff
@@ -713,6 +713,11 @@ engines:
     engine : 1337x
     shortcut : 1337x
     disabled : True
+    
+  - name : Duden
+    engine : duden
+    shortcut : duden
```

Comment:
Why not have a shorter shortcut? "du" is not yet taken.

Reply:
Ok, I can change that.